# CalibrationFeatures Message

std_msgs/Header header

std_msgs/UInt32 frameId

std_msgs/Float32[] p0
std_msgs/Float32[] p1
std_msgs/Int8[] status



# struct CalibrationFeatures {
#   frameId @0 :UInt32;

#   p0 @1 :List(Float32);
#   p1 @2 :List(Float32);
#   status @3 :List(Int8);
# }
